rules/chewing-the-fat/rule.md

```diff
@@ -0,0 +1,80 @@
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Luke Cook
+    url: https://www.ssw.com.au/people/luke-cook
+  - title: Piers Sinclair
+    url: https://www.ssw.com.au/people/piers-sinclair
+  - title: Alex Rothwell
+    url: https://www.ssw.com.au/people/alex-rothwell
+created: 2023-03-29T23:46:59.734Z
+guid: 1b4cea4e-f968-4b75-8d1d-44c3aa781fd5
+---
+No matter how well adapted your business and processes are, there's always room to improve and learn.
+Getting feedback from others, outside the [critical distance](/importance-critical-distance), helps to fine-tune ideas, even if you may think your processes are ideal.
+
+Once you've built a fantastic team of experts, it's important to actively listen to their opinions and ideas to continue building a great place to work and deliver more value to your customers.
+
+<!--endintro-->
+
+In the highly dynamic space of software development, there's new frameworks, features and languages constantly being released, hyped, maturing and sunsetting.
+No-one has the time to keep up with everything that's going on, so it makes sense to gauge the opinions of your team - there's a good chance that at least one of them has an opinion!
+
+Setting up a regular schedule to collect everyone's opinion is great to build up a habit and ensure that you've always got new ideas coming in.
+
+# Chewing The Fat
+
+Gathering and actioning these opinions is a relatively straight-forward process which can be referred to as "Chewing The Fat" (CTF).
+While it may be tempting to stick to only business oriented topics, it's a good idea to remember that this process works well to spread knowledge around your team and upskill everyone!
+
+A typical CTF cycle has 4 core steps:
+
+1. Publish an article on a new technology or process that you'd like to explore as a team
+2. Send out a survey to everyone on the team, asking them to review the article and give their feedback
+3. Review the results and implement feedback
+4. Publish changes
+
+## 1. Publish an article
+
+If you're excited about a shiny new technology or process change that you've read about, choosing a topic for the article will be easy.
+Sometimes it may be a bit more difficult, this is where your team could contribute ideas that they want discussed.
+
+Write up an article and publish it somewhere appropriate for your team to access.
+Make sure to include [bad and good examples](/use-bad-and-good-examples) to really get your point across.
+
+## 2. Send out a survey
+
+Once the article is published, send out a survey to your team and set a deadline for submissions.
+Without a deadline, it's difficult to start the review process as responses can come in well after you've moved on to the next CTF cycle.
+
+In the survey, ask for the team's thoughts on the idea presented in the article, both positive and negative criticism should be encouraged - the point is to choose the best ideas, bad ideas should be rejected.
```

Comment:
This is a looong sentence. Break it up a little. E.g.,

In the survey, ask for the team's thoughts on the idea presented in the article. Both positive and negative criticism should be encouraged - the point is to choose the best ideas, bad ideas should be rejected.